code/modules/hydroponics/grown/plum.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/seeds/plum
+	name = "pack of plum seeds"
+	desc = "These seeds grow into plum trees."
+	icon_state = "seed-plum"
+	species = "plum"
+	plantname = "Plum Tree"
+	product = /obj/item/food/grown/plum
+	lifespan = 55
+	endurance = 35
+	growthstages = 6
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_grow = "plum-grow"
+	icon_dead = "plum-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	mutatelist = null
```

Comment:
```suggestion

```
you can delete this, the parent has the same value

Reply:
Same as above.